core/Command/Background/Mode.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2015 Christian Kampka <christian@kampka.net>
+ * SPDX-License-Identifier: MIT
+ */
+namespace OC\Core\Command\Background;
+
+use OCP\IAppConfig;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Mode extends Command {
+	public function __construct(
+		private IAppConfig $appConfig,
+	) {
+		parent::__construct();
+	}
+
+	protected function configure(): void {
+		$this
+			->setName('background:cron')
+			->setAliases(['background:ajax', 'background:webcron'])
+			->setDescription('Use cron, ajax or webcron to run background jobs');
+	}
+
+	protected function execute(InputInterface $input, OutputInterface $output): int {
+		$mode = match ($input->getArgument('command')) {
+			'background:cron' => 'cron',
+			'background:ajax' => 'ajax',
+			'background:webcron' => 'webcron',
+		};
```

Comment:
```suggestion
				default => 'ajax',
		};
```

Maybe to make psalm happy, but not sure if this is the best way todo so.

Reply:
Thanks :+1: 

I overlooked the psalm warning. 
Fixed it by adding a phpdoc.